scripts/install_third_party.py

```diff
@@ -200,10 +202,10 @@ def return_json(filepath):
     """Return json object when provided url
 
     Args:
-        filepath: the path to the json file.
+        filepath: str. The path to the json file.
 
     Returns:
-        a parsed json objects.
+        json. A parsed json object.
```

Comment:
json is not a valid typeinfo, please fix. Remember  that the typeinfo is what is printed when you call type() on the variable.